### ISSUE TITLE
[SUGGESTION] To have even more compat with Upgrade Aquatic.

### ISSUE THREAD

**Issue by KKotik** [ItsBlackGear/CavesAndCliffsBackport#19]
It whould be cool to have config option to replace CCB glow squid spawns with Upgrade Aquatic ones. Why? Many mods still using UA's glow inc, also you can buckit them and they have cool animated texture.

**Comment by StrangerSon71** [comment]
Yeah, I need my dirt cups. https://docs.google.com/document/d/1AKKKaOurkT2nYLBqnqu6W7NS6j0vk1tHVPyKpPKbfcY/edit

**Comment by Toma400** [comment]
Does CCB spawn replace UA one, or do they spawn both?
If the latter, then I'd request for that being disabled by default, since I really love CCB glow squids :3

**Comment by ItsBlackGear** [comment]
we don't replace UA glow squids, but have a feature to disable our squids if you want to use UA ones 👀
